magefiles/test.go

```diff
@@ -48,6 +48,20 @@ func (Test) Integration() error {
 	return goTest("./internal/services/integrationtesting/...", "-tags", "ci,docker", "-timeout", "15m")
 }
 
+// Steelthread Run steelthread tests
+func (Test) Steelthread() error {
```

Comment:
Add to `Test.All`?

Reply:
Related: I don't see any CI config for this. Where and how often will we run these?

Reply:
The idea would be to run them manually, either via mage here or a manual action in CI